### DISCUSSION TITLE
코드 하이라이트 적용이 안됩니다.

### DISCUSSION THREAD

**Discussion by SeungGyu-Park** [pronist/hELLO#73] [Q&A] [answered]
**환경 (버그가 발생한 환경을 적어주세요.):**

- 운영체제: Windows 10
- 브라우저: chrome
- 버전: 3.6.1
- 스킨 버전 (스킨 버전은 index.xml 에서 확인): [e.g 3.4.0]

**설명**
버그가 어떻게 발생하는지 알려주세요. 대략적인 버그의 설명을 적어주시면 됩니다.
[e.g. 코드 하이라이팅이 동작하지 않습니다.]

스킨 변경을 새로 세번 시도했는데 매번 안됐습니다.
제가 잘못한건지 다른분들은 이런 현상을 겪으신분이 안계시네요.
좋은 스킨 잘 활용하지 못해서 속상하네요 ㅠㅠ

코드블럭 highlight적용이 안됩니다.
cdn으로도 해보고 파일 업로드해서 올렸는데
배경색이랑 겹쳐서 그러는지 색이 이상하게 나오네요.
css에서 pre code의 배경을 강제로 바꿔도 코드 색이 이상하게 적용됩니다.

**발생 과정**
버그가 어떤 과정을 통해 발생하는지 구체적으로 적어주세요.
[e.g. 테마 변경 버튼을 통해 화이트 모드로 변경시 코드 하이라이팅의 색상이 변하지 않습니다. ...]

하이라이트 적용했는데 하이라이팅 색상이 적용은 되는거 같은데 배경이 안바뀌어서 그런지 이상하게 나옵니다.
플러그인은 처음부터 안썼습니다.

코드 하이라이팅
**스크린샷 및 링크**
버그가 발생한 상황을 가능하다면 스크린샷을 첨부해주세요. 또한 버그가 발생한 모습이 있는 포스트의 주소를 첨부해주시는 것도 아주 좋습니다!
![1](https://user-images.githubusercontent.com/89517818/157890550-7bfd1293-7684-47a9-bbc0-41488cf02125.PNG)
![image](https://user-images.githubusercontent.com/89517818/157890668-38ded234-0bf7-4327-8601-172e3833331d.png)

https://sg-dev.tistory.com/79

**Comment by pronist** [comment, marked as answer]
안녕하세요.

스킨에 코드 하이라이트는 이미 포함되어 별도로 포함할 필요가 없습니다. 또한 테마를 포함하는 기본적인 코드또한 포함되어 있어 `스킨 옵션` 에서 테마 이름만 넣으면 됩니다.

이 질문은 중복 질문으로 생각됩니다. #42 를 참고해주세요.

**Comment by SeungGyu-Park** [comment]
아.... 3시간동안 해결 못했던게... 간단했군요 감사합니다.
추가로 혹시 다크모드로 변경할 때 h1태그와 quote 인용구 부분 텍스트가 하얀색으로 변하지 않는데 어떻게 할 수 있나요?
https://sg-dev.tistory.com/79
여기 보시면 다크모드일때 h1태그와 quote 부분 박스가 검정색으로 나옵니다.

**Comment by pronist** [comment]
@seunggyu-park 혹시 블로그 작성하실 때 마크다운을 사용해서 작성하시나요? 마크다운의 경우에는 현재 #52 에서 이야기 되었던 것 처럼 스킨 차원에서의 스타일링을 지원하지 않습니다. `티스토리 기본 에디터` 에서는 제목 1이 `h2` 로 취급되기 때문에 `h1` 은 처리되지 않았습니다. 추후 마크다운 관련 패치를 진행할 때가 되면 처리 될 예정입니다.

**Comment by SeungGyu-Park** [comment]
@pronist 그렇군요.. alt + shift + 1~4 로 마크다운을 쓰고 있습니다.
감사합니다! 혹시 인용구 박스 부분도 아직은 패치가 안된 것일까요?

**Comment by pronist** [comment]
@seunggyu-park 인용문구의 경우 `티스토리 기본 에디터` 에서 `blockquote[data-ke-style="style3"]` 와 같은 형태로 처리되기 때문에 `blockquote` 자체에 대해서는 스타일링 처리되지 않았습니다.

+ 작성하신 글을보니 검은색으로 처리되어있으신데, HTML 편집을 켜서 보시면 `blockquote` 내부에 `span` 이 사용되어 강제적으로 인라인 스타일링 처리가 되어있네요. 그 부분이 문제라면 `span` 태그 문제입니다.

**Comment by SeungGyu-Park** [comment]
@pronist 그렇군요 다크모드에서도 보일 수 있는 박스형태로 정리할 수 있는 코드를 설정하려면 어떻게 해야 할까요?

**Comment by pronist** [comment]
@seunggyu-park `blockquote` 내부에서 인라인으로 스타일링 된 `span` 태그를 제거해보시기 바랍니다.

**Comment by SeungGyu-Park** [comment]
해결되었습니다. 감사합니다!
